### ISSUE TITLE
Add support for AVIF image format

### ISSUE THREAD

**Issue by BugBuster1701** [BugBuster1701/contao-banner-bundle#53]
- [ ] first Contao / Imagine: [Ticket](https://github.com/contao/contao/issues/2283)
- [ ] check support in getimagesize / exif_imagetype: [Constants](https://www.php.net/manual/en/function.exif-imagetype.php#refsect1-function.exif-imagetype-constants)
- [ ] As an alternative via Extension: `strtolower(pathinfo($imageFilename, \PATHINFO_EXTENSION));`

**Comment by BugBuster1701** [comment]
Test an test picture: https://libre-software.net/avif-test/

**Comment by BugBuster1701** [comment]
* getimagesize: 
```
IMG_AVIF (int)
Used as a return value by [imagetypes()](https://www.php.net/manual/en/function.imagetypes.php) 
Available as of PHP 8.1.0.
```

**Comment by BugBuster1701** [comment]
`'avif' => \defined('IMAGETYPE_AVIF') ? IMAGETYPE_AVIF : 19,`
aus https://github.com/contao/contao/blob/4.13/core-bundle/src/Resources/contao/library/Contao/File.php#L270
